examples/mptcp.cfg

```diff
@@ -0,0 +1,21 @@
+# You can test this configuration by running the command:
+#
+#   $ mptcpize run curl localhost:5000
+
+global
+   strict-limits  # refuse to start if insufficient FDs/memory
+   # add some process-wide tuning here if required
+
+defaults
+   mode http
+   balance roundrobin
+   timeout client 60s
+   timeout server 60s
+   timeout connect 1s
+
+frontend main
+    bind mptcp@[::]:5000
+    default_backend mptcp_backend
+
+backend mptcp_backend
+    server mptcp_server mptcp@[::]:4331
```

Comment:
it might be good to add something like:

    # MPTCP is usually used on the frontend, but it is also possible to enable it to communicate with the backend

(+ strip after ~80 chars)